packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -118,14 +120,15 @@ export class DataSyncPlugin extends GraphbackPlugin {
   protected addDataSyncFieldsToModel(schemaComposer: SchemaComposer<any>, model: ModelDefinition) {
     const name = model.graphqlType.name;
     const modelTC = schemaComposer.getOTC(name);
+    const TimestampSTC = schemaComposer.getSTC(GraphbackTimestamp.name);
 
     modelTC.addFields({
       [DataSyncFieldNames.lastUpdatedAt]: {
-        type: GraphbackTimestamp
+        type: TimestampSTC.getType()
```

Comment:
I am not 100% sure, but you can just reference the scalar type directly.

See [`7402981` (#1771)](https://github.com/aerogear/graphback/pull/1771/commits/7402981623195f2a86203d1ed7c1b5cd1e17d341#r465712478)